package.json

```diff
@@ -67,7 +67,8 @@
   "private": true,
   "resolutions": {
     "//": "See https://github.com/facebook/create-react-app/issues/11773",
-    "react-error-overlay": "6.0.9"
+    "react-error-overlay": "6.0.9",
+    "@types/react": "17.0.2"
```

Comment:
Why you don't bump this line https://github.com/tdex-network/tdex-dashboard/blob/4710f4dd41f459f03738c791d28d3061aa6d06f3/package.json#L49
instead of resolution ?